src/Wallabag/CoreBundle/Form/Type/ConfigType.php

```diff
@@ -9,16 +9,20 @@
 class ConfigType extends AbstractType
 {
     private $themes = array();
+    private $languages = array();
 
     /**
-     * @param array $themes Themes come from the LiipThemeBundle (liip_theme.themes)
+     * @param array $themes    Themes come from the LiipThemeBundle (liip_theme.themes)
+     * @param array $languages Languages come from configuration
```

Comment:
Could you add more information about what should be `$languages` (an array with code as key and label as value)